src/views/Home/Home.jsx

```diff
@@ -158,10 +163,15 @@ const Home = () => {
 			{doneLoading && hasResults ? (
 				<div className="results">
 					<h3>{`${i18n.resultsFor[language]}: ${keyword}`}</h3>
-					{showDefinition && <Definition {...stateDefinitionResult} />}
-					{showBestBet && (
-						<BestBet language={language} results={stateBestBetResult} />
-					)}
+					<div
```

Comment:
Following BEM let's treat this as a modifier.  So it would be that we have .results__feature--bestbet--definition vs results__feature--bestbet or results__feature--definition.
It does complicate things stylesheet-wise.